app/helpers/application_helper.rb

```diff
@@ -1,2 +1,23 @@
 module ApplicationHelper
 end
+
+class ActionView::Helpers::FormBuilder
```

Comment:
That sounds alright, what about `{lib,app}/patches`, `{lib,app}/extensions`, `{lib,app}/enhancements` to emphasise the fact you're extending a standard class/module?